pkg/util/xorm/session_convert.go

```diff
@@ -17,6 +17,11 @@ import (
 	"xorm.io/core"
 )
 
+const (
+	zeroTime0 = "0000-00-00 00:00:00"
+	zeroTime1 = "0001-01-01 00:00:00"
```

Comment:
Are these used?

Reply:
yes in this file, i just deleted the time_helper, since all other things inside is not used.